.github/scripts/install_crd_rbac.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# This script will create a ClusterRoleBinding that grants the
+# default serviceAccount the cluster-admin role so that CRs can be
+# created and fetched by the tekton task tests.
+#
+# Requirements:
+# - Connection to a running k8s cluster (e.g. kind)
+
+# The tekton task tests run as the default serviceAccount
+cat << EOF > crd_rbac.yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pipeline-test-role
+  namespace: default
+rules:
+- apiGroups:
+  - appstudio.redhat.com
+  resources:
+  - releases
+  - releaseplans
+  - releaseplanadmissions
+  - releasestrategies
+  - snapshots
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: pipeline-test-binding
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: pipeline-test-role
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
+EOF
+kubectl apply -f crd_rbac.yaml
```

Comment:
Have you considered running this in the workflow and have a yaml file instead of a bash script? I guess it's not a big difference. But by having a raw yaml file, you can take advantage of yaml highlighting and formatting in IDE when you work with it.

Reply:
Also, is this really needed? Just to be sure. I would assume that since we're running Kind with an admin user, it would have unlimited permissions. Normally that's how it works. Maybe that's not the case with crds?

Reply:
Yes, it is needed. The pipelineruns are executed as the default service account. That service account does not have admin privileges

Reply:
I'm not really sure what you mean by your first suggestion. Just commit the yaml file and then have the action `kubectl apply -f` the file? Do you have an example of what you mean?

Reply:
> I'm not really sure what you mean by your first suggestion. Just commit the yaml file and then have the action kubectl apply -f the file? Do you have an example of what you mean?

Yes, that's what I meant. And now I see David also suggested to have the yaml files separate.

What I suggest is just have this in the workflow:
```yaml
      - name: Install RBAC for CRDs
        run: kubectl apply -f .github/resources/crd_rbac.yaml
```
`.github/resources/crd_rbac.yaml` is a file that would contain what you create in line in your script in the current version.

Reply:
> Yes, it is needed. The pipelineruns are executed as the default service account. That service account does not have admin privileges

Makes sense, thanks.

Reply:
changed to a yaml file

Reply:
Thanks.